package.json

```diff
@@ -23,6 +23,7 @@
     "repl": "npm run build && node utils/replInit.ts",
     "test:integration:node": "TEST_ENV=node jest src/integration/ -c jest.config.integration-node.js --runInBand --bail",
     "test:integration:edge": "TEST_ENV=edge jest src/integration/ -c jest.config.integration-edge.js --runInBand --bail",
+    "test:integration-local:node": "node src/integration/integrationRunner.js",
```

Comment:
For local runs

Reply:
Can we update the existing `test:integration:node` and `test:integration:edge` commands to handle calling `integrationRunner` rather than creating new calls that also run the tests with the proper setup and teardown?